easybuild/easyconfigs/d/DAAL/DAAL-2017.1.132.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = "IntelBase"
+name = 'DAAL'
+version = '2017.1.132'
+
+homepage = 'http://software.intel.com/en-us/intel-daal'
+description = """Boost machine learning and big data analytics performance 
+ with easy-to-use library.
+ Features highly tuned functions for deep learning, classical 
+ machine learning, and data analytics performance across spectrum 
+ of Intel architecture devices.
+ Optimizes data ingestion together with algorithmic computation for 
+ highest analytics throughput.
+ Includes Python*, C++, and Java* APIs and connectors to popular data 
+ sources including Spark* and Hadoop*.
+ Free and open source community-supported versions are available, as 
+ well as paid versions that include premium support."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['l_%(name)s_%(version)s.tgz']
```

Comment:
@shahzebsiddiqui Please include a `SHA256` checksum (you can use `eb --inject-checksums` for this)